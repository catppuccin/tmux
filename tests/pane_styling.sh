#!/usr/bin/env bash

script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
# shellcheck disable=SC1091
source "${script_dir}/helpers.sh"

tmux set -g @catppuccin_pane_status_enabled "yes"

tmux source "${script_dir}/../catppuccin_options_tmux.conf"
tmux source "${script_dir}/../catppuccin_tmux.conf"

print_option pane-border-format

# Switch the number position to the right
tmux set -g @catppuccin_pane_number_position "right"
tmux source "${script_dir}/../catppuccin_tmux.conf"
print_option pane-border-format

tmux set -g @catppuccin_pane_number_position "left" # reset

# Fill option "all"
tmux set -g @catppuccin_pane_default_fill "all"
tmux source "${script_dir}/../catppuccin_tmux.conf"
print_option pane-border-format

tmux set -g @catppuccin_pane_default_fill "number" # reset

# Fill option "none"
tmux set -g @catppuccin_pane_default_fill "none"
tmux source "${script_dir}/../catppuccin_tmux.conf"
print_option pane-border-format
