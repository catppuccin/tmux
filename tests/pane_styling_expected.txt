pane-border-format #[fg=#a6e3a1,bg=default]█#[fg=#313244,bg=#a6e3a1]#{pane_index}#[fg=#a6e3a1,bg=#313244]█ #[fg=#a6e3a1,bg=#313244]#{b:pane_current_path}#[fg=#313244,bg=default]█
pane-border-format #[fg=#313244,bg=default]█#[fg=#a6e3a1,bg=#313244]#{b:pane_current_path} #[fg=#a6e3a1,bg=#313244]█#[fg=#313244,bg=#a6e3a1]#{pane_index}#[fg=#a6e3a1,bg=default]█
pane-border-format #[fg=#a6e3a1,bg=default]█#[fg=#313244,bg=#a6e3a1]#{pane_index}#[fg=#a6e3a1,bg=#313244]█ #[fg=#313244,bg=#a6e3a1]#{b:pane_current_path}#[fg=#a6e3a1,bg=default]█
pane-border-format #[fg=#313244,bg=default]█#[fg=#cdd6f4,bg=#313244]#{pane_index}#[fg=#cdd6f4,bg=#313244]█ #[fg=#cdd6f4,bg=#313244]#{b:pane_current_path}#[fg=#313244,bg=default]█
